t/mime-header.t

```diff
@@ -136,6 +136,7 @@ while (my ($e, $d) = splice @splice, 0, 2) {
     is Encode::decode('MIME-Header', $e) => $d, info("decode default: $e => $d");
 }
 
+no warnings 'once';
```

Comment:
That is wrong fix! I'm preparing new patches for Encode::MIME::Header which also fix this warning. Please drop this one change so as it is incorrect and will also cause conflicts with my future changes into Encode::MIME::Header. Thanks!